src/library/Text/styled.js

```diff
@@ -47,14 +47,22 @@ export const TextRoot = styled('p', {
     ['color', 'fontWeight'].indexOf(prop) === -1 && isPropValid(prop)
 })(
   ({
+    /* TargetX Custom Styles */
+    altFont,
+    bold,
+    fontFamily,
+    fontSize,
+    lineHeight,
+
+    /* Built-In Styles */
     align,
     appearance: propAppearance,
     as,
     color,
     defaultAppearance,
     fontWeight,
     inherit,
-    noMargins,
+    noMargins = true,
```

Comment:
On second thought, let's just remove this default setting `= true` for now.  Unnecessary diff since we're going to set `Text_marginBottom = 0` anyway. Also, I think ultimately, we'll end up support `marginTop` and `marginBottom` at some point.